system-tests/fixtures/api/TestApi/package.json

```diff
@@ -11,7 +11,7 @@
     "message": {
       "moduleName": "TestApi",
       "namespace": "com.TestApi.ern",
-      "apiSchemaPath": "/Users/blemair/Code/electrode-native/system-tests/fixtures/api/testapi-schema.json",
+      "apiSchemaPath": "/Users/f0b00n7/github/electrode-io/electrode-native/system-tests/fixtures/api/testapi-schema.json",
```

Comment:
pls, check if this is intentional. IMHO, none is correct, is CI workspace set up as /Users/blemair/Code/?

Reply:
Yeah, I also don't like this. Check #1301